indico/web/client/js/jquery/extensions/global.js

```diff
@@ -9,6 +9,13 @@
 /* global showFormErrors:false */
 /* eslint-disable max-len */
 
+import moment from 'moment';
```

Comment:
Not sure if this is the right file to put this..

Reply:
Does it also work if you put it in indico/web/client/js/jquery/index.js? Because in there we have all the other stuff like importing the moment locales...